docs/reference/errors-and-warnings/NU5046.md

```diff
@@ -0,0 +1,58 @@
+---
+title: NuGet Error NU5046
+description: NU5046 Error code
+author: dominoFire
+ms.author: feaguila
+ms.date: 9/4/2019
+ms.topic: reference
+ms.reviewer: karann
+f1_keywords: 
+  - NU5046
+---
+
+# NuGet Error NU5046
+
+<pre>The icon file 'Icon.png' does not exist in the package.</pre>
+
+
+# Issue
+
+NuGet is unable find the local file that is marked as the package icon.
```

Comment:
Calling it a `local file` makes it sound like it's looking on the filesystem, not in the package, but the error text in the `<pre>` block, and most of the solution makes it sound like the error is when the icon is not in the package.

How about:
```suggestion
NuGet is unable find the icon file in the package.
```